.github/workflows/ci.yml

```diff
@@ -25,33 +24,34 @@ jobs:
       - name: Lint
         run: pnpm lint
       - name: Run Tests
-        run: pnpm test:ember
+        run: pnpm test
 
   floating:
     name: "Floating Dependencies"
     runs-on: ubuntu-latest
-    timeout-minutes: 10
 
     steps:
       - uses: actions/checkout@v4
       - uses: wyvox/action-setup-pnpm@v3
         with:
           node-version: 18
-          args: '--no-lockfile'
```

Comment:
I think we need this because otherwise it is not testing with floating dependencies.